### ISSUE TITLE
Reopening discussion for: `padding-line-between-statements` Add an option for padding object elements

### ISSUE THREAD

**Issue by johndebord** [eslint/eslint#16251]
> Unfortunately, it looks like there wasn't enough interest from the team
> or community to implement this change. While we wish we'd be able to
> accommodate everyone's requests, we do need to prioritize. We've found
> that issues failing to reach accepted status after 21 days tend to
> never be accepted, and as such, we [close those issues](https://eslint.org/docs/maintainer-guide/issues#when-to-close-an-issue).
> This doesn't mean the idea isn't interesting or useful, just that it's
> not something the team can commit to.
> 
> Thanks for contributing to ESLint and we appreciate your understanding.
> 
> [//]: # (auto-close)
> 
> _Originally posted by @eslint-deprecated[bot] in https://github.com/eslint/eslint/issues/12948#issuecomment-620361209_

Having a padding option for more than just statements would be convenient.
e.g.,
```javascript
// Error:
myFunc(var
  .split("")
  .reverse()

  .join("")
  .replace(/[0-9]/, "x")
);

// Correct:
myFunc(var
  .split("")
  .reverse()
  .join("")
  .replace(/[0-9]/, "x")
);
```

```javascript
// Error:
<MyComponent
  myProp={{
    myNestedProp0: 0,

    myNestedProp1: 1,
  }}
/>

// Correct:
<MyComponent
  myProp={{
    myNestedProp0: 0,
    myNestedProp1: 1,
  }}
/>
```

```javascript
// Error:
<MyComponent
  myProp0=0

  myProp1=1
/>

// Correct:
<MyComponent
  myProp0=0
  myProp1=1
/>
```

Currently, there is no such rule (that I know of) that enforces this behavior

**Comment by mdjermanovic** [comment]
Hi @johndebord, thanks for the issue!

Per our rules policies, [stylistic rules are frozen](https://github.com/eslint/eslint#stylistic-rule-updates), so we are not adding new options to the `padding-line-between-statements` rule. You can always copy the existing rule into a custom rule and make it behave the way you want.

**Comment by johndebord** [comment]
Hi @mdjermanovic, I wasn't aware that the rules were frozen. Thanks for the quick response & feedback
